src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -64,8 +61,8 @@ void UsgsAstroFrameSensorModel::reset() {
     m_ccdCenter = std::vector<double>(2, 0.0);
     m_spacecraftVelocity = std::vector<double>(3, 0.0);
     m_sunPosition = std::vector<double>(3, 0.0);
-    m_odtX = std::vector<double>(10, 0.0);
-    m_odtY = std::vector<double>(10, 0.0);
+    m_distortionType = DistortionType::TRANSVERSE;
+    m_opticalDistCoeffs.clear();
```

Comment:
This should default to 20 0s. Right now, the transverse distortion method would seg-fault because it expects 20 coefficients.